.travis.yml

```diff
@@ -0,0 +1,63 @@
+language: python
+sudo: false
+cache: pip
+
+python:
+  - "2.7"
+
+addons:
+  apt:
+    packages:
+      - expect-dev  # provides unbuffer utility
+      - python-lxml  # because pip installation is slow
+      - python-simplejson
+      - python-serial
+      - python-yaml
+      - python-pypdf  # repo specific
+
+env:
+  global:
+  - VERSION="7.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  #- TRANSIFEX_USER='transbot@odoo-community.org'
```

Comment:
You can remove all this part, but you need the secure line. Check here what is the corresponding one: https://github.com/OCA/maintainer-quality-tools/issues/194